c2rust-transpile/src/lib.rs

```diff
@@ -1,3 +1,17 @@
+#![allow(
+    clippy::borrowed_box,
+    clippy::duplicate_underscore_argument,
+    clippy::enum_variant_names,
+    clippy::large_enum_variant,
+    clippy::match_ref_pats,
+    clippy::new_without_default,
+    clippy::single_match,
+    clippy::too_many_arguments,
+    clippy::try_err,
+    clippy::type_complexity,
+    clippy::useless_format
+)]
```

Comment:
Same as the other blanket `#![allow(...)]`: either `#[allow(...)]` them on a specific item with a comment explaining why, or leave them as warnings to be fixed by less-trivial changes.